vassal-app/src/main/java/VASSAL/build/widget/ListWidget.java

```diff
@@ -54,16 +55,42 @@
   private JPanel multiPanel;
   private int width, height, divider;
   private static final String DIVIDER = "divider";
+  public static final String SCALE = "scale"; //$NON-NLS-1$
+  protected double scale;
 
   private Map<Object,String> keys = new HashMap<>();
   private int count = 0;
 
   public ListWidget() {
+    scale = 1.0;
   }
 
   public static String getConfigureTypeName() {
     return "Scrollable List";
   }
+  
+  @Override
+  public boolean hasScale() {
+    return true;
+  }
+  
+  @Override
+  public double getScale() {
+    return scale;
+  }  
+  
+  // True if we're part of main (docked) Piece Palette
+  protected boolean isMainPiecePalette() {
+    Widget w = this;
+    while (w.getParent() != null) {
+      w = w.getParent();
```

Comment:
Here's one more I assignment in `while` condition you could do that I missed the first time.

Reply:
Taken care of